src/hdiff_stencil_variant.h

```diff
@@ -20,6 +20,7 @@ namespace platform {
 
         std::vector<std::string> stencil_list() const override;
 
+        void prerun_init() override;
```

Comment:
Why do we need this? Can’t this work be done in the constructor?

Reply:
We don't. It was easier to test the first-touch NUMA stuff this way. I moved it to the constructor.